src/PlayerAccount/PlayerAccount.php

```diff
@@ -104,7 +104,7 @@ public function useRequestInsurance(bool $shouldUseRequestInsurance = true): sel
      *
      * @return Response|RequestInsurance
      *
-     * @throws PlayerAccountRequestFailedException
+     * @throws PlayerAccountRequestFailedException|BindingResolutionException
```

Comment:
Kan vi ikke bruge multiline throws statements istedet? :) 

```suggestion
     * @throws PlayerAccountRequestFailedException
     * @throws BindingResolutionException
```

Reply:
Det kan vi godt :)